tests/gfx/DrawStringViaImageTest.java

```diff
@@ -0,0 +1,49 @@
+package gfx;
+
+import javax.microedition.lcdui.Canvas;
+import javax.microedition.lcdui.Display;
+import javax.microedition.lcdui.Graphics;
+import javax.microedition.lcdui.Image;
+import javax.microedition.midlet.MIDlet;
+import com.nokia.mid.ui.*;
+
+public class DrawStringViaImageTest extends MIDlet {
```

Comment:
Sweet, this is a very necessary test given the way that WhatsApp in particular draws text